packages/tooling/cfp-template/template/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Welcome } from "./components";
+
```

Comment:
I learned recently that ES6 modules have an import syntax to [import side effects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_a_module_for_its_side_effects_only) (such as defining an element). I don't know if it will work w/ webpack minification, but we could potentially avoid this "// prevent tree shaking" pattern by taking that approach. Might be worth a try?

Reply:
Oh, neato that works, fixed!